user_guide/intro_company_structure_org_selector.rst

```diff
@@ -0,0 +1,198 @@
+.. _user-guide-getting-started-company-structure:
+
+Company Structure and Organization Selector
+===========================================
+
+Understanding the Company Structure
+-----------------------------------
+
+Each enterprise is structured in a meaningful way. Big enterprises usually have several branches/offices, each of which 
+may be divided into several major divisions comprised of departments that may be comprised of several functional groups, 
+etc. Even small companies organize their workforce into teams.
+
+For example, a telecommunication service provider might have several subsidiaries, e.g. in each of the states. The 
+Californian subsidiary may have two offices - Western and Eastern. In each of the offices there are unites working at 
+different directions:
```

Comment:
Comment: I'm not sure if this example is based on an actual case or not, but if it is hypothetical, it would make more geographical sense to divide California by north/south rather than west/east.